scripts/check_compliance.py

```diff
@@ -236,11 +236,15 @@ def parse_codeowners(self, git_root, codeowners):
                         if abs_path.endswith("/"):
                             abs_path = abs_path + "**"
                         elif os.path.isdir(abs_path):
-                            error("Wrong syntax: {}".format(abs_path))
+                            self.case.result = Failure(
+                                "Expected / after directory in CODEOWNERS: "
+                                + abs_path)
                             continue
+
                         g = glob.glob(abs_path, recursive=True)
                         if not g:
-                            error("Path does not exist: {}".format(path))
+                            self.case.result = Failure(
+                                "Path does not exist: " + path)
```

Comment:
In other examples Failure() takes two arguments, why not here?

Reply:
Didn't spot it. Fixed.